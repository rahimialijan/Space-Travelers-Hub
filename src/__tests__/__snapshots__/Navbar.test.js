import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom'; // Import BrowserRouter
import NavbarComponent from '../../components/navbarComponents';

test('NavbarComponent renders correctly', () => {
  const { container } = render(
    <BrowserRouter>
      {' '}
      {/* Wrap NavbarComponent with BrowserRouter */}
      <NavbarComponent />
    </BrowserRouter>,
  );
  expect(container).toMatchSnapshot();
});
