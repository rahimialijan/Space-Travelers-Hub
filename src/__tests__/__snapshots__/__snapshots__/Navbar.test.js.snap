// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarComponent renders correctly 1`] = `
<div>
   
  <div
    class="navbar"
  >
    <div
      class="logo-container"
    >
      <a
        aria-current="page"
        class="logo-items active"
        href="/"
      >
        <img
          alt="Logo"
          class="planet-logo"
          src="planet.png"
        />
        <h1
          class="logo-name"
        >
           Space Travelers Hub
        </h1>
      </a>
    </div>
    <div
      class="navbar-links"
    >
      <ul>
        <li>
          <a
            aria-current="page"
            class="nav-link active"
            href="/"
          >
            Rockets
          </a>
        </li>
        <li>
          <a
            class="nav-link"
            href="/mission"
          >
            Missions
          </a>
        </li>
        <li
          class="horzental-line"
        >
          |
        </li>
        <li>
          <a
            class="nav-link"
            href="/myprofile"
          >
            My Profile
          </a>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
